docs/labwc-theme.5.scd

```diff
@@ -122,6 +128,14 @@ labwc-config(5).
 	Width of a titlebar button, in pixels.
 	Default is 26.
 
+*window.button.spacing*
+	Space between titlebar buttons, in pixels.
+	Default is 0.
+
+*window.button.hover.bg.shape*
+	The shape of the hover effect of a titlebar button: "rectangle" or "circle".
```

Comment:
May be “squared” and “rounded” instead?

Reply:
I think circle and rectangle are good words